tests-web/www/js/main.js

```diff
@@ -197,13 +189,14 @@ class TestCase {
 
 function importOSM(groupName, app, osmXML, drivingSide, addOSMLayer) {
   try {
+    const importSettings = app.getImportSettings();
     const network = new JsStreetNetwork(osmXML, {
       driving_side: drivingSide,
-      debug_each_step: app.importSettings.debugEachStep,
-      dual_carriageway_experiment: app.importSettings.dualCarriagewayExperiment,
+      debug_each_step: !!importSettings.debugEachStep,
```

Comment:
The "double bang" is a JS idiom for "cast to float". It's simply a double "not".